images/linux/scripts/installers/1604/powershellcore.sh

```diff
@@ -8,6 +8,7 @@
 source $HELPER_SCRIPTS/document.sh
 
 LSB_RELEASE=$(lsb_release -rs)
```

Comment:
Where do we use this variable?

Reply:
variable has been removed.
commit: https://github.com/andy-mishechkin/virtual-environments/commit/aca1c5af1c1e99ca7f4164a20720554ec738b558